apps/user-service/pyproject.toml

```diff
@@ -9,7 +9,7 @@ license = "MIT"
 python = "^3.9"
 pika = "^1.3.1"
 peewee = "^3.15.4"
-psycopg2-binary = "^2.9.5"
+psycopg2 = "^2.9.5"
```

Comment:
add this as a hard dependency to allow mac to build it instead of getting an incompatible binary locally